odk1-src/form-question-types.rst

```diff
@@ -906,30 +862,27 @@ specify an :th:`appearance` of :tc:`compact-2`.
   abcd_icon,c,C,c.jpg
   abcd_icon,d,D,d.jpg
 
-.. _quickcompact-widget:
 
-Compact select widget with autoadvance
-""""""""""""""""""""""""""""""""""""""""
+.. _select-no-buttons-widget:
+
+Select widget with no-buttons appearance
+""""""""""""""""""""""""""""""""""""""""""""
 
 type
   :tc:`select_one {list_name}`
 appearance
-  :tc:`quickcompact`
-
-The :tc:`quickcompact` appearance
-adds :ref:`autoadvance <autoadvance>` functionality
-to the design of the :ref:`compact-image-select`.
+  :tc:`no-buttons`
 
-.. video:: /vid/form-widgets/quickcompact.mp4
+When the :tc:`no-buttons` appearance is added, the app does not display buttons but images or labels if images are missing.
```

Comment:
"the app does not display buttons but images or labels if images are missing." -> "the app displays choice labels and/or images without the selection radio button. This is particularly useful for building a grid of images."

Reply:
`And` shouldn't be used here. We display images if media files are provided or labels otherwise never both. Right?

Reply:
Ah! Yes, I think you're right. That's not what I got from your original sentence. Something like "the app displays choices without the selection radio button. If images are specified for choices, only the images are displayed. This is particularly useful for building a grid of images."

Reply:
Ok, fixed.